### ISSUE TITLE
Matthew Landau

### ISSUE THREAD

**Issue by zpellish** [MarchingLizard/brifony#38]
Adam,

Matt Landau, a staff member, completed both a staff and participant application. Birthright just canceled his participant application today, however when I log in as him in the Application Status Page it's a participant portal without him being on the trip. We need Matt to be seeing the staff portal with him placed on the trip, and so he can enter his information attached to the staff application.

**Comment by zpellish** [comment]
Adam,

I wanted to follow up on this and see if there is any new information on this?

**Comment by IshaDakota** [comment]
@zpellish I'm not quite getting this can we talk later today?

**Comment by zpellish** [comment]
Yeah we can talk later today, let me know a time that works for you.

On Wed, May 11, 2016 at 8:24 AM, Adam Stewart <notifications@github.com>
wrote:

> @zpellish <https://github.com/zpellish> I'm not quite getting this can we
> talk later today?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/MarchingLizard/brifony/issues/38#issuecomment-218457692>
>



-- 
Zach Pellish
North American Operations Manager
Shorashim
Main line: 312-267-0677
Direct line: 312-380-6536

*SIGN UP for **Birthright Israel: Shorashim* <http://bit.ly/1USYrzo>
*for winter 2016/2017 or summer 2017!*
